src/libraries/formatUtils/dataFormatting.ts

```diff
@@ -14,9 +15,15 @@ export const opdDataFormatter = (
     diseases && data.disease3 !== ""
       ? diseases.find((el) => el.code === +data.disease3)
       : null;
```

Comment:
There is a reason why are you using a unary plus operator so frequently inside this function?

Reply:
We have to map each **disease code** gives by the form to the correspondant object.

Reply:
The disease returned by formik is just a string code, we have to convert it to the entire object in order to call the api. That is why I am using the + operator to convert that string to integer in order to find the right object.